src/CommandBasedBackendRegisterAccessor.cc

```diff
@@ -214,7 +309,21 @@ namespace ChimeraTK {
       throw ChimeraTK::runtime_error("Device not functional when reading " + this->getName());
     }
```

Comment:
This was here before, but I am wondering why. The check should have been done in preWrite(), and I think it actually is done there in the framework (TransferElement?).